src/main/scala/net/snowflake/spark/snowflake/io/StageWriter.scala

```diff
@@ -319,6 +319,23 @@ private[io] object StageWriter {
     }
   }
 
+  private[snowflake] def getStageTableName(tableName: String): String = {
+    def genTableName():String =
+      s"spark_stage_table_${System.currentTimeMillis()}_${Math.abs(Random.nextInt)}"
+
+    val tableNameQuoted = "(.*)\"([^\"]*)\"(\\s*)".r
+    val tableNameNotQuoted = "(.*)\\.([^.]*)".r
+
+    tableName match {
+      case tableNameQuoted(databaseSchemaPart, _, _) =>
+        s"$databaseSchemaPart${genTableName()}"
+      case tableNameNotQuoted(databaseSchemaPart, _) =>
+        s"$databaseSchemaPart.${genTableName()}"
+      case _ =>
```

Comment:
how about this name `DB."schema".TABLE`?

Reply:
The table name will be generated as `DB."schema".spark_stage_table_XXX_YYY`

Unit test covers below:
My_DB."My.Schema"."Table.1" ---> My_DB."My.Schema".spark_stage_table_1605311023640_940705254

Reply:
`aa.`?